skbuild/resources/cmake/FindF2PY.cmake

```diff
@@ -0,0 +1,98 @@
+#.rst:
+#
+# This module defines CMake functions to build Python extension modules and
+# stand-alone executables.
```

Comment:
Would it make sense to tweak the description based on text found at http://www.f2py.com/  ?